pytorch_lightning/utilities/device_parser.py

```diff
@@ -29,6 +29,12 @@ def determine_root_gpu_device(gpus: List[int]) -> Optional[int]:
 
     Returns:
         designated root GPU device id
+
+    Raises:
+        TypeError:
+            If GPUs are not in a list
```

Comment:
```suggestion
            If ``gpus`` is not a List
```

though i don't get why we're raising an exception if the typehint already lists that this should be List[int]

Reply:
type hint doesn't enforce so...